Formula/gst-plugins-rs.rb

```diff
@@ -22,14 +21,19 @@ class GstPluginsRs < Formula
   depends_on "ninja" => :build
   depends_on "rust" => :build
   depends_on "dav1d"
+  depends_on "gst-plugins-bad" # for gst-webrtc
   depends_on "gst-plugins-base"
   depends_on "gstreamer"
   depends_on "gtk4"
+  depends_on "libpthread-stubs"
   depends_on "pango" # for closedcaption
+  depends_on "python@3.11" # for tomli
 
-  # commit ref, https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/commit/ea98a0b5964cd196abbb48c621969a8ef33eb157
-  # remove in next release
-  patch :DATA
+  # used by meson dependency checker to parse cargo files
+  resource "tomli" do
+    url "https://files.pythonhosted.org/packages/c0/3f/d7af728f075fb08564c5949a9c95e44352e23dee646869fa104a3b2060a3/tomli-2.0.1.tar.gz"
+    sha256 "de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f"
+  end
```

Comment:
I don't think there's a usage of this resource in the formula

Reply:
Yep, looks like it uses tomllib by default and only falls back to tomli for older Pythons. Fixed in 777be6d892f0c752aecdf6814127b13314e33d7e. ty!